.ci/jobs/apm-aws-lambda-smoketests-mbp.yml

```diff
@@ -0,0 +1,44 @@
+---
+- job:
+    name: library/apm-aws-lambda-smoketests-mbp
+    display-name: AWS Lambda extension Smoke Tests
+    description: AWS Lambda extension Smoke Tests
+    project-type: multibranch
+    concurrent: true
+    script-path: .ci/smoke-tests.groovy
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: false
+        head-filter-regex: '(main|PR-.*|v\d+\.\d+\.\d+.*)'
+        notification-context: 'apm-ci'
+        repo: apm-aws-lambda
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - regular-branches: true
+        - change-request:
+            ignore-target-only-changes: true
+        clean:
+          after: true
+          before: true
+        prune: true
+        shallow-clone: true
+        depth: 3
+        do-not-fetch-tags: true
+        submodule:
+          disable: false
+          recursive: true
+          parent-credentials: true
+          timeout: 100
+        timeout: '15'
+        use-author: true
+        wipe-workspace: true
+        property-strategies:
+          all-branches:
+          - suppress-scm-triggering: true
```

Comment:
This section will avoid listening for SCM events, such as git merge commits or github commands, is that the expected behaviour?

Reply:
> I've moved this file to another PR. Good question on the commit/PRs run.

Let me know when I can review it. The configuration for the `.ci/jobs/*.yml` might require some further analysis depending when those jobs should be ran, as I pointed earlier, happy to discuss further :)

Reply:
Opened a new PR and removed the option :+1: